lib/plugins/bidirectionalData.js

```diff
@@ -208,22 +208,22 @@ function processTransformationsNGSIv2(values, transformations, callback) {
         values.push({
             name: resultTransformations[i].object_id,
             type: resultTransformations[i].type,
-            value: parser.parse(cleanedExpression, ctx, 'String')
+            value: parser.parse(cleanedExpression, ctx, 'String', typeInformation)
         });
     }
 
     callback(null, values);
 }
 
 /**
- * Apply a list of transformations to the reported values, 
+ * Apply a list of transformations to the reported values,
  * generating a new array of values with the additional data.
  * For NGSI-LD the output includes value, metadata and datasetId
  *
  * @param {Array} values                List of reported attributes (with their name, type and value).
  * @param {Array} transformations       List of transformations to apply (with their name, type and expression).
  */
-function processTransformationsNGSILD(values, transformations, callback) {
+function processTransformationsNGSILD(values, transformations, typeInformation, callback) {
```

Comment:
`typeInformation` should be added as new `@param` in the function "JavaDoc"

Reply:
Fixed e0220ae